scripts/snaptron/process_tp53_junctions.py

```diff
@@ -37,16 +38,42 @@ def get_rail_id(row):
             all_sample_ids.append(sample_id)
     return(all_sample_ids)
 
-# File paths
-sample_file = 'samples.tsv.gz'
-junction_file = 'tp53_junctions.txt.gz'
-mut_scores_file = os.path.join('..', '..', 'classifiers', 'TP53',
-                               'tables', 'mutation_classification_scores.tsv')
-out_junc_file = 'junctions_with_mutations.csv.gz'
+parser = argparse.ArgumentParser()
+
+parser.add_argument('--sample_file', default=None,
+                    help='Filename of SNAPTRON samples')
+parser.add_argument( '--junction_file', default=None,
+                    help='Filename of SNAPTRON junctions')
+parser.add_argument( '--alt_folder', default=None,
+                    help='Folder containing classifier')
+parser.add_argument( '--output', default=None,
+                    help='Filename of output junctions')
+
+args = parser.parse_args()
 
-samples = pd.read_table(sample_file, compression='gzip', low_memory=False)
+
+# File paths
+sample_file = args.sample_file or 'samples.tsv.gz'
+junction_file = args.junction_file or 'tp53_junctions.txt.gz'
+if args.alt_folder:
+    mut_scores_file = os.path.join(args.alt_folder,
+                                   'tables', 'mutation_classification_scores.tsv')
+else:
+    mut_scores_file = os.path.join('..', '..', 'classifiers', 'TP53',
+                                   'tables', 'mutation_classification_scores.tsv')
+out_junc_file = args.output or 'junctions_with_mutations.csv.gz'
+# FIXME: upgrading to pandas 0.24.0+ will give 'infer' for to_csv
```

Comment:
again, link to #100 

will this line break the current pipeline without upgrading?